server/src/core/utilities.ts

```
@@ -194,24 +197,30 @@ function validateCustomGameSettings(
     };
   }
   let ok = true;
-  let errors = [];
+  const errors = [];
   for (let key in settings) {
```

Biome

[error] This let declares a variable that is only assigned once.

'key' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)